### PR TITLE
Icons for qv2ray

### PR DESCRIPTION
**Pull request by palob**
https://github.com/lhy0403/Qv2ray

v2ray_1024
v2ray_1024-ci
v2ray_1024-sq

### CODE REVIEW

**Comment by maxtron95** [comment]
@palob The program has a new website and icon just to let you know since you posted this.

Website: https://github.com/Qv2ray/Qv2ray
Icon: 
qv2ray

The Papirus Icon I worked on for example: [Gv2ray Icon](https://github.com/PapirusDevelopmentTeam/papirus-icon-theme/blob/master/Papirus/64x64/apps/qv2ray.svg)

**Comment by palob** [comment]
Turns out my reference icon wasn't the old qv2ray one but the current v2ray (which qv2ray is a graphical frontend to) one.

What's the new one supposed to show? That's not even abstract art.

**Comment by maxtron95** [comment]
@palob I can't tell what that is or what they were trying to represent on that one.

I thought it was some weird bird wing or whatever.

The icon I put there is the icon they are actually using in their github repo and flathub.

The link you put up seems like they are redirecting it to the one I put up.

**Comment by ghost** [comment]
I am the Qv2ray maintainer. 

The icon which @palob uploaded a month ago was our design of the old v1 icon, and what's more, that was actually the logo of our upstream, V2ray (https://v2ray.com). 

Our (Qv2ray, the frontend of V2ray) **old** logo was a `Q` covering that purple image. As can be found [here](https://github.com/Qv2ray/Qv2ray-v1/blob/master/icons/Qv2ray.png)

However, weeks ago we have published our Qv2ray version 2 and changed our logo, you can find it [here](https://github.com/Qv2ray/Qv2ray/blob/master/assets/icons/qv2ray.png)

If anyone wants to use Qv2ray's icon, please use the latter one. Thanks!

**Comment by DuckSoft** [comment]
This is @DuckSoft from [Qv2ray](https://github.com/Qv2ray). 

Sorry to have you guys confused about those two icons. To clarify, the icon referenced by @palob is the one we used for *old v1 version*, whose *maintenance is stopped* now. We are now working on v2 version branch, and decided to **use the icon referenced by @creepertron95**.

Thank you for your efforts!

**Comment by ghost** [comment]
And please, the old link https://github.com/lhy0403/Qv2ray is now deprecated, please use https://github.com/Qv2ray/Qv2ray instead, for package Id, please use the `com.github.Qv2ray` only. Thanks

**Comment by DuckSoft** [comment]
> I thought it was some weird bird wing or whatever.

You are correct. That's a **parrot** in our new icon. Qv2ray is mainly used for breaking the limitation of Great Firewall of China (or virtually any limitations on network). The bird here symbolizes **freedom**.

>图片
> Picture from Bing. Public Domain. Just for Reference. :smile:

**Comment by palob** [comment]
Thanks for chiming in, will give the new one a shot when I find time.

**Comment by palob** [comment]
Pushed these new designs

QV2RAY-ci
QV2RAY-sq

**Review by DuckSoft — APPROVED** [review]
LGTM. Thanks for your great work!

**Comment by ghost** [comment]
that looks cool dude, thanks for your hard work!

**Review by ghost — CHANGES_REQUESTED** [review]
Well the old ID if Qv2ray is now removed, so I think it’s unnecessary to put it here.

File: data.json [hunk 1]
```diff
@@ -15757,6 +15757,16 @@
             "root": "qutim"
         }
     },
+    "qv2ray": {
+        "linux": {
+            "root": "qv2ray",
+            "symlinks": [
+                "com.github.lhy0403.Qv2ray",
```
Comment (ghost) [inline data.json]: This one can be safely removed FYI

**Comment by palob** [comment]
Sorry, which line exactly are you referring to?

**Comment by ghost** [comment]
> Sorry, which line exactly are you referring to?

the `com.github.lhy0403.Qv2ray` one can be removed, thanks

**Review by Foggalong — APPROVED** [review]

**Pull request merged**